functions/stats.js

```diff
@@ -31,22 +31,30 @@ exports.getOnlineStats = functions.https.onCall(async (data, context) => {
   const venues = await admin.firestore().collection("venues").get();
   await Promise.all(
     venues.docs.map(async (venue) => {
-      let venueOpen = false;
-      await venue.ref
-        .collection("events")
-        .get()
-        .then((events) => {
-          events.docs.forEach((event) => {
-            if (eventIsNow(event, now)) {
-              venueOpen = true;
+      const template = venue.data().template;
+      if (template === "zoomroom") {
+        let venueOpen = false;
+        await venue.ref
+          .collection("events")
+          .get()
+          .then((events) => {
+            events.docs.forEach((event) => {
+              if (eventIsNow(event, now)) {
+                venueOpen = true;
+              }
+            });
+            if (venueOpen) {
+              const venueWithId = venue.data();
+              venueWithId.id = venue.id;
+              openVenues.push(venueWithId);
             }
           });
-          if (venueOpen) {
-            const venueWithId = venue.data();
-            venueWithId.id = venue.id;
-            openVenues.push(venueWithId);
-          }
-        });
+      }
+      if (template === "artpiece" || template === "themecamp") {
```

Comment:
I would change this to be an `else` clause to act as a default for all the other cases.